src/components/applicationDetailsContainer.tsx

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+function ApplicationDetailsContainer() {
+  return (
+    <div className="flex flex-col gap-y-8 ">
```

Comment:
important: the sidebar grey background must not expand to the column size, it must fit to the size of the content of itself.